.yarnrc

```diff
@@ -0,0 +1 @@
+pure-lockfile true
```

Comment:
Why?

Reply:
Don't create a lock file, but allow the developer to use one if present.